l10n_it_ddt/wizard/ddt_create_invoice.xml

```diff
@@ -8,8 +8,9 @@
             <field name="arch" type="xml">
               <form string="DdT Create Invoice">
                   <group>
-                    <field name="journal_id" domain="[('type', 'in', ['sale', 'sale_refund'])]"/>
```

Comment:
@sergiocorato teoricamente, grazie alle record rules, l'utente dovrebbe già essere in grado di vedere solo i sezionali della propria azienda.
Ti riferisci al caso in cui l'utente è admin?